test/unit/lifecycle.js

```diff
@@ -4,38 +4,66 @@ import helpers from '../lib/helpers';
 import skate from '../../src/skate';
 
 describe('Lifecycle Callbacks', function () {
-  it('should call the created() callback when the element is attached', function (done) {
-    var tagName = helpers.safeTagName('my-element');
-    skate(tagName.safe, {
+  var MyEl;
+  var myEl;
```

Comment:
Hmm, I'm generally not a fan of var names that are this similar. Would you consider `myEl` and `myElConstructor` (or `myElCtor`)?